admin_guide/router.adoc

```diff
@@ -10,45 +10,47 @@
+password and port were configured during the router installation, but can be
+found by viewing the *_haproxy.conf_* file on the container.
 
 == Viewing Logs
-To view a router log, simply run the `oc log` command for the pod.  Keep in mind that since
-the router is running as a plugin process that manages the underlying implementation the log
-that you will see is the log for the plugin, not the actual HAProxy log.  
+To view a router log, run the `oc log` command on the pod. Note that, since the
+router is running as a plug-in process that manages the underlying
+implementation, the log is for the plug-in, not the actual HAProxy log.
 
 == Router Internals
 .Routes.json
 
-As routes are processed by the HAProxy router they are stored both in memory, on disk, and in
-the HAProxy configuration file.  In `/var/lib/containers/router/routes.json` you can find the
-internal representation of the routes that is passed to the template to generate the HAProxy
-configuration file.  When troubleshooting a route issue it may help to look at this file to
-see the data that is being used to drive configuration.
+Routes are processed by the HAProxy router, so they are stored both in memory,
+on disk, and in the HAProxy configuration file. You can find the internal
+representation of the routes that is passed to the template to generate the
+HAProxy configuration file in the *_/var/lib/containers/router/routes.json_*
+file. When troubleshooting a routing issue, view this file to see which data is
+being used to drive configuration.
 
 .HAProxy Config
 
-In `/var/lib/haproxy/conf/haproxy.conf` you can find the configuration that is being used
-by HAProxy.  Here is where you can view the backends that have been created for specific routes.
-Also, in the same directory you can find the mapping files that are used by the helper 
-frontend and backends when mapping an incoming request to a backend.
+You can find the HAProxy configuration and the backends that have been created
+for specific routes in the *_/var/lib/haproxy/conf/haproxy.conf_* file. The
+mapping files used by the helper frontend and backends when mapping an incoming
+request to a backend are found in the same directory.
 
 .Certificates
 
-Certificates are stored in two places. Certificates that are presented at the edge for edge
-terminated and re-encrypt terminated routes are stored in `/var/lib/containers/router/certs/`.
-Certificates that are used when connecting to backends for re-encrypt terminated routes are
-stored in `/var/lib/containers/router/cacerts/`.  The files are keyed by the namespace and
-name of the route.  You will find that the key, certificate, and CA certificate are 
-concatenated into a single file.  You may use openssl to view the contents of these files.
-
-
+Certificates are stored in two places. Certificates that are presented at the
+edge for edge terminated and re-encrypt terminated routes are stored in the
```

Comment:
The "edge for edge terminated" part loses me a bit. Could you put part of the sentence in parentheses? Like this:

Certificates that are presented at the edge (for edge terminated and re-encrypt terminates routes) are stored in the /var/lib/containers/router/certs directory.